pyrk/materials/flibe.py

```diff
@@ -6,6 +6,8 @@
 class Flibe(Material):
     """This class represents FLiBe. It inherits from the material
     class and possesses attributes intrinsic to flibe.
+    All properties from the report: Temperature-Dependent Thermophysical Properties
+    for Fluoride Salts and Simulant Fluids[TH lab at UCB]
```

Comment:
This comment still stands. We need a public, citable, reference for the value of these thermal property changes you've made.